rfc/007-workload-secrets.md

```diff
@@ -0,0 +1,108 @@
+# RFC 007: Workload secrets
+
+## Objective
+
+Provide Contrast workloads with secret key material, such that:
+
+1. Workloads can use key material to derive additional secrets autonomously.
+2. The key material is stable across workload upgrades.
+3. The key material is stable across manifest generations.
+4. The key material is stable across Contrast upgrades.
```

Comment:
Should we also use this secret key material to derive the private keys for our TLS CA?

Reply:
We already do for the root CA, and the mesh CA is deliberately random.